CONTRIBUTING.md

```diff
@@ -55,7 +55,7 @@ cd mlrun
 
 Create a conda environment and activate it
 ```shell script
-conda create -n mlrun python=3.9
+conda create -n mlrun python=3.9 protobuf
```

Comment:
why isnt part of `install-conda-requirements` then?

Reply:
Done